package.json

```
@@ -49,11 +67,15 @@
     "acorn"
   ],
   "scripts": {
+    "unit": "npm-run-all -s unit:*",
+    "unit:esm": "nyc mocha --color --reporter progress --timeout 30000 'tests/lib/**/*.js'",
+    "unit:cjs": "mocha --color --reporter progress --timeout 30000 tests/lib/commonjs.cjs",
     "generate-regex": "node tools/generate-identifier-regex.js",
-    "test": "npm run-script lint && node Makefile.js test",
-    "lint": "node Makefile.js lint",
-    "fixlint": "node Makefile.js lint --fix",
-    "sync-docs": "node Makefile.js docs",
+    "test": "npm-run-all -p unit lint",
+    "lint": "eslint lib/ espree.js .eslintrc.cjs tests/lib/",
+    "fixlint": "eslint --fix lib/ espree.js .eslintrc.cjs tests/lib/",
```

```suggestion
    "lint": "eslint \"*.?(c)js\" lib/ tests/lib/",
    "fixlint": "npm run lint -- --fix",
```

To lint `rollup.config.js` and `sync-docs.js`, too.

(they currently have some linting errors, most of them can be autofixed)